rse-project.Rmd

```diff
@@ -692,6 +692,53 @@ Breaking this down:
 > Documentation that serves these two groups well
 > will almost certainly serve the needs of strangers as well.
 
+These additional rules are taken from [@Good2014,@Mich2015,@Hart2016,@Zook2017]:
+
+-   **Keep raw data raw.**
+    Exactly what constitutes "raw" data isn't always clear:
+    the analog inputs to a voltmeter probably don't need to be recorded on tape
+    if we have the voltmeter's digitized output.
+    However,
+    our analysis methods and tools can change over time,
+    and even after years of work they can contain bugs.
+    We should therefore keep the original data exactly as we received it
+    so that we can re-do previous analyses or try new ones.
+    This is also a good safeguard if anyone ever questions the integrity of the research:
+    it will be hard to defend ourselves against accusations of fraud
+    if we can't show our work.
+
+-   **Give every part of the data a unique identifier.**
+    To aid reproducibility (and save a lot of confusion in conversation),
+    it should be possible to uniquely identify every part of every dataset.
+    The dataset itself should have a [DOI][doi] (Section \@ref(rse-publish-identifiers)),
+    and within the dataset,
+    every file should have a meaningful name (Section \@ref(rse-project-data-vs-code)).
+    Going further,
+    every record should have a unique [key][key] within the dataset (Section \@ref(rse-publish-data)).
+
+-   **Have a backup plan.**
+    Every hard drive eventually fails;
+    the hardware needed to read various storage media eventually becomes unavailable,
+    and there is always the risk that bankruptcy, corporate merger, or a court order
+    will shut down the site we've been using to store our data.
+    We should therefore always have a second copy in a different physical and jurisdictional locational.
+    If the primary store is GitHub,
+    copy the data onto a CD-ROM and give it to the departmental secretary for safekeeping.
```

Comment:
If I tried to do this I would get massive pushback because their filing systems are not designed for data nor is my data their responsibility to ensure.  Many academic institutions have some form of institutional repository where things can be uploaded to, some long term data storage solution, or a unit better equipped to handle long term data storage requests than front office admin staff.  Also, some cheaper brands of burnable CD-ROMs only have a lifespan of 1-2 years.  People should first contact their departmental level IT department to ask for policies, then expand their searches to their library, research IT group, or grant compliance staff as needed.